streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -107,6 +107,15 @@ class StreamingContext private[streaming] (
    */
   def this(path: String) = this(path, new Configuration)
 
+
+  def this(path: String, sparkContext: SparkContext) = {
```

Comment:
This should probably have scaladoc to explain that it restores from a checkpoint?